wrapcheck/wrapcheck.go

```diff
@@ -91,11 +101,15 @@ func NewAnalyzer(cfg WrapcheckConfig) *analysis.Analyzer {
 
 func run(cfg WrapcheckConfig) func(*analysis.Pass) (interface{}, error) {
 	// Precompile the regexps, report the error
-	ignoreSigRegexp, err := compileRegexps(cfg.IgnoreSigRegexps)
+	ignoreSigRegexp, err1 := compileRegexps(cfg.IgnoreSigRegexps)
+	ignoreInterfaceRegexps, err2 := compileRegexps(cfg.IgnoreInterfaceRegexps)
```

Comment:
I was thinking, I've actually made some similar changes, still local for now, in order to precompile the globs. Could we maybe update this to something along the lines of:

```go
var (
	ignoreSigRegexp    []*regexp.Regexp
	ignorePackageGlobs []glob.Glob
	err                error
)

ignoreSigRegexp, err = compileRegexps(cfg.IgnoreSigRegexps)
if err != nil {
	ignorePackageGlobs, err = compileGlobs(cfg.IgnorePackageGlobs)
}
```

I will likely refactor this anyway in an upcoming PR, but for now I'd prefer not to have multiple error checks in the returned func below.